backend/api/Areas/Admin/Controllers/UserController.cs

```diff
@@ -82,9 +83,12 @@ public IActionResult GetUsers(EModel.UserFilter filter)
         /// <returns>Paged object with an array of users.</returns>
         [HttpGet("/api/admin/my/users")]
         public IActionResult GetMyUsers(int page = 1, int quantity = 10, string sort = null) // TODO: sort and filter.
```

Comment:
I'd recommend using the same arguments as the `GetUsers(EModel.UserFilter filter)` possibly.  Then you have the ability to apply additional filters and sorting built in.